validation-model/src/main/java/org/verapdf/gf/model/GFModelParser.java

```diff
@@ -123,10 +125,17 @@ private static PDFAFlavour obtainFlavour(PDDocument document) {
 			VeraPDFMeta veraPDFMeta = VeraPDFMeta.parse(is);
 			Integer identificationPart = veraPDFMeta.getIdentificationPart();
 			String identificationConformance = veraPDFMeta.getIdentificationConformance();
+			String name = "";
+			if (identificationPart == null && identificationConformance == null) {
```

Comment:
@bdoubrov @BezrukovM is it a good idea to introduce PDF/UA logic into PDF/A specific class? Wouldn't it be more proper to introduce another implementation and interface for PDF/UA parsing? 
I assuming that logically PDF/UA is NOT a PDF/A, is it?